quarkus-observability-demo-activemq/quarkus-observability-demo-activemq-legume/src/main/java/com/brunobat/rest/resource/LegumeResource.java

```diff
@@ -33,68 +33,84 @@ public class LegumeResource implements LegumeApi {
     @Inject
     MessageSender messageSender;
 
-    @Transactional
-    public Response provision() {
+    @WithTransaction
```

Comment:
Why Panache's @Tx instead of plain Jakarta @Tx ?

Reply:
Reactive code requires a different annotation. I got a warning message.